core/services/job/runner_integration_test.go

```diff
@@ -162,18 +162,35 @@ func TestRunner(t *testing.T) {
 	})
 
 	t.Run("referencing a non-existent bridge should error", func(t *testing.T) {
-		_, bridge := cltest.MustCreateBridge(t, db, cltest.BridgeOpts{}, config)
+		// Create a random bridge name
+		_, b := cltest.MustCreateBridge(t, db, cltest.BridgeOpts{}, config)
+		// Reference a different one
 		jb := makeOCRJobSpecFromToml(t, fmt.Sprintf(`
 			type               = "offchainreporting"
 			schemaVersion      = 1
 			observationSource = """
-				ds1          [type=bridge name="%s"];
+				ds1          [type=bridge name="blah"];
 			"""
-		`, bridge.Name.String()))
+		`))
+		// Should error creating it
 		err := jobORM.CreateJob(jb)
-		require.Error(t,
```

Comment:
interestingly this test was invalidated in https://github.com/smartcontractkit/chainlink/commit/a89ffb596dd0a5a266d12b22ff6a9d3886e361d5#diff-973acc7e5085b3bee58aa8b6bfd76c7efcd748c404049c28a1ba0fa622c9d5b5R176, but since this assertion was broken (should have actually been a require.Equal) it silently kept working